### PR TITLE
Change of base image on oberthur/docker-ubuntu:16.04

### PR DESCRIPTION
**Pull request by mpschmidt83**

### CODE REVIEW

File: example/k8s/cluster/redis1-rc.yaml [hunk 1]
```diff
@@ -21,7 +21,7 @@ spec:
     spec:
       containers:
       - name: redis-node
-        image: mpschmidt/elastic-redis:3.2.5-1.0
+        image: mpschmidt/elastic-redis:3.2.5-1.1
```
Comment (dawidmalina) [inline example/k8s/cluster/redis1-rc.yaml]: Can you rename this to oberthur

File: example/k8s/cluster/redis2-rc.yaml [hunk 2]
```diff
@@ -21,7 +21,7 @@ spec:
     spec:
       containers:
       - name: redis-node
-        image: mpschmidt/elastic-redis:3.2.5-1.0
+        image: mpschmidt/elastic-redis:3.2.5-1.1
```
Comment (dawidmalina) [inline example/k8s/cluster/redis2-rc.yaml]: Can you rename this to oberthur

File: example/k8s/cluster/redis3-rc.yaml [hunk 3]
```diff
@@ -21,7 +21,7 @@ spec:
     spec:
       containers:
       - name: redis-node
-        image: mpschmidt/elastic-redis:3.2.5-1.0
+        image: mpschmidt/elastic-redis:3.2.5-1.1
```
Comment (dawidmalina) [inline example/k8s/cluster/redis3-rc.yaml]: Can you rename this to oberthur

File: example/k8s/cluster/redis4-rc.yaml [hunk 4]
```diff
@@ -21,7 +21,7 @@ spec:
     spec:
       containers:
       - name: redis-node
-        image: mpschmidt/elastic-redis:3.2.5-1.0
+        image: mpschmidt/elastic-redis:3.2.5-1.1
```
Comment (dawidmalina) [inline example/k8s/cluster/redis4-rc.yaml]: Can you rename this to oberthur

File: example/k8s/cluster/redis5-rc.yaml [hunk 5]
```diff
@@ -21,7 +21,7 @@ spec:
     spec:
       containers:
       - name: redis-node
-        image: mpschmidt/elastic-redis:3.2.5-1.0
+        image: mpschmidt/elastic-redis:3.2.5-1.1
```
Comment (dawidmalina) [inline example/k8s/cluster/redis5-rc.yaml]: Can you rename this to oberthur

File: example/k8s/cluster/redis6-rc.yaml [hunk 6]
```diff
@@ -21,7 +21,7 @@ spec:
     spec:
       containers:
       - name: redis-node
-        image: mpschmidt/elastic-redis:3.2.5-1.0
+        image: mpschmidt/elastic-redis:3.2.5-1.1
```
Comment (dawidmalina) [inline example/k8s/cluster/redis6-rc.yaml]: Can you rename this to oberthur

File: example/k8s/sentinel/redis-node1-rc.yaml [hunk 7]
```diff
@@ -18,7 +18,7 @@ spec:
     spec:
       containers:
       - name: redis-node
-        image: mpschmidt/elastic-redis:3.2.5-1.0
+        image: mpschmidt/elastic-redis:3.2.5-1.1
```
Comment (dawidmalina) [inline example/k8s/sentinel/redis-node1-rc.yaml]: Can you rename this to oberthur

File: example/k8s/sentinel/redis-node2-rc.yaml [hunk 8]
```diff
@@ -18,7 +18,7 @@ spec:
     spec:
       containers:
       - name: redis-node
-        image: mpschmidt/elastic-redis:3.2.5-1.0
+        image: mpschmidt/elastic-redis:3.2.5-1.1
```
Comment (dawidmalina) [inline example/k8s/sentinel/redis-node2-rc.yaml]: Can you rename this to oberthur

File: example/k8s/sentinel/redis-node3-rc.yaml [hunk 9]
```diff
@@ -18,7 +18,7 @@ spec:
     spec:
       containers:
       - name: redis-node
-        image: mpschmidt/elastic-redis:3.2.5-1.0
+        image: mpschmidt/elastic-redis:3.2.5-1.1
```
Comment (dawidmalina) [inline example/k8s/sentinel/redis-node3-rc.yaml]: Can you rename this to oberthur

File: example/k8s/sentinel/redis-sentinel1-rc.yaml [hunk 10]
```diff
@@ -21,7 +21,7 @@ spec:
     spec:
       containers:
       - name: redis-sentinel
-        image: mpschmidt/elastic-redis:3.2.5-1.0
+        image: mpschmidt/elastic-redis:3.2.5-1.1
```
Comment (dawidmalina) [inline example/k8s/sentinel/redis-sentinel1-rc.yaml]: Can you rename this to oberthur

File: example/k8s/sentinel/redis-sentinel2-rc.yaml [hunk 11]
```diff
@@ -21,7 +21,7 @@ spec:
     spec:
       containers:
       - name: redis-sentinel
-        image: mpschmidt/elastic-redis:3.2.5-1.0
+        image: mpschmidt/elastic-redis:3.2.5-1.1
```
Comment (dawidmalina) [inline example/k8s/sentinel/redis-sentinel2-rc.yaml]: Can you rename this to oberthur

File: example/k8s/sentinel/redis-sentinel3-rc.yaml [hunk 12]
```diff
@@ -21,7 +21,7 @@ spec:
     spec:
       containers:
       - name: redis-sentinel
-        image: mpschmidt/elastic-redis:3.2.5-1.0
+        image: mpschmidt/elastic-redis:3.2.5-1.1
```
Comment (dawidmalina) [inline example/k8s/sentinel/redis-sentinel3-rc.yaml]: Can you rename this to oberthur

File: example/k8s/standalone/redis-rc.yaml [hunk 13]
```diff
@@ -18,7 +18,7 @@ spec:
     spec:
       containers:
       - name: redis-node
-        image: mpschmidt/elastic-redis:3.2.5-1.0
+        image: mpschmidt/elastic-redis:3.2.5-1.1
```
Comment (dawidmalina) [inline example/k8s/standalone/redis-rc.yaml]: Can you rename this to oberthur

File: example/k8s/cluster/redis1-rc.yaml [hunk 14]
```diff
@@ -21,7 +21,7 @@ spec:
...
         imagePullPolicy: IfNotPresent
         command: ["/run.sh"]
```
Comment (dawidmalina) [inline example/k8s/cluster/redis1-rc.yaml]: Why do you need to override command to exactly the same default value?

File: Dockerfile [hunk 15]
```diff
@@ -1,11 +1,33 @@
...
+
+
+
+RUN mkdir /data
+VOLUME /data
+WORKDIR /data
+
+EXPOSE 6379
 
 CMD [ "/run.sh" ]
```
Comment (dawidmalina) [inline Dockerfile]: I think this could be just an ENTRYPOINT

File: Dockerfile [hunk 16]
```diff
@@ -1,11 +1,33 @@
...
+ENV REDIS_VERSION 3.2.5
+
 COPY run.sh /run.sh
-RUN chown redis:redis /run.sh && \
-    chmod +x /run.sh
+
+RUN chmod +x run.sh \
+    && apt-get update \
+    && apt-get install -y build-essential wget ruby \
+    && wget -O redis.tar.gz http://download.redis.io/releases/redis-$REDIS_VERSION.tar.gz \
```
Comment (dawidmalina) [inline Dockerfile]: Change to curl which is already in base image

File: Dockerfile [hunk 17]
```diff
@@ -1,11 +1,33 @@
...
+    && mkdir -p /usr/src/redis \
+    && tar -xzf redis.tar.gz -C /usr/src/redis --strip-components=1 \
+    && rm redis.tar.gz \
+    && grep -q '^#define CONFIG_DEFAULT_PROTECTED_MODE 1$' /usr/src/redis/src/server.h \
+    && sed -ri 's!^(#define CONFIG_DEFAULT_PROTECTED_MODE) 1$!\1 0!' /usr/src/redis/src/server.h \
+    && grep -q '^#define CONFIG_DEFAULT_PROTECTED_MODE 0$' /usr/src/redis/src/server.h \
+    && make -C /usr/src/redis \
+    && make -C /usr/src/redis install \
+    && rm -r /usr/src/redis
+
```
Comment (dawidmalina) [inline Dockerfile]: There should be also cleaning part
```
    # clean up
    && apt-get clean autoclean \
    && apt-get autoremove --yes \
    && rm -rf /var/lib/{apt,dpkg,cache,log}/
```

File: example/k8s/cluster/redis1-rc.yaml (hunk 1 above)
Comment (mpschmidt83) [inline example/k8s/cluster/redis1-rc.yaml]: I will change when official image will be ready

File: example/k8s/cluster/redis2-rc.yaml (hunk 2 above)
Comment (mpschmidt83) [inline example/k8s/cluster/redis2-rc.yaml]: I will change when official image will be ready

File: example/k8s/cluster/redis3-rc.yaml (hunk 3 above)
Comment (mpschmidt83) [inline example/k8s/cluster/redis3-rc.yaml]: I will change when official image will be ready

File: example/k8s/cluster/redis4-rc.yaml (hunk 4 above)
Comment (mpschmidt83) [inline example/k8s/cluster/redis4-rc.yaml]: I will change when official image will be ready

File: example/k8s/cluster/redis5-rc.yaml (hunk 5 above)
Comment (mpschmidt83) [inline example/k8s/cluster/redis5-rc.yaml]: I will change when official image will be ready

File: example/k8s/cluster/redis6-rc.yaml (hunk 6 above)
Comment (mpschmidt83) [inline example/k8s/cluster/redis6-rc.yaml]: I will change when official image will be ready

File: example/k8s/sentinel/redis-node1-rc.yaml (hunk 7 above)
Comment (mpschmidt83) [inline example/k8s/sentinel/redis-node1-rc.yaml]: I will change when official image will be ready

File: example/k8s/sentinel/redis-node2-rc.yaml (hunk 8 above)
Comment (mpschmidt83) [inline example/k8s/sentinel/redis-node2-rc.yaml]: I will change when official image will be ready

File: example/k8s/sentinel/redis-node3-rc.yaml (hunk 9 above)
Comment (mpschmidt83) [inline example/k8s/sentinel/redis-node3-rc.yaml]: I will change when official image will be ready

File: example/k8s/sentinel/redis-sentinel1-rc.yaml (hunk 10 above)
Comment (mpschmidt83) [inline example/k8s/sentinel/redis-sentinel1-rc.yaml]: I will change when official image will be ready

File: example/k8s/sentinel/redis-sentinel2-rc.yaml (hunk 11 above)
Comment (mpschmidt83) [inline example/k8s/sentinel/redis-sentinel2-rc.yaml]: I will change when official image will be ready

File: example/k8s/sentinel/redis-sentinel3-rc.yaml (hunk 12 above)
Comment (mpschmidt83) [inline example/k8s/sentinel/redis-sentinel3-rc.yaml]: I will change when official image will be ready

File: example/k8s/standalone/redis-rc.yaml (hunk 13 above)
Comment (mpschmidt83) [inline example/k8s/standalone/redis-rc.yaml]: I will change when official image will be ready

**Pull request merged**